examples/android-add-flutter-to-existing-app-example/README.md

```diff
@@ -0,0 +1,109 @@
+  final String USER_ID = "yourUsername@yourCompanyName"; // when integrating with Interaction Studio use a numeric user id - see https://eu2.thunderhead.com/one/help/interaction-studio/how-do-i/mobile/one_integrate_mobile_find_integration_info/#username-user-id
+  final String HOST = "https://xx.thunderhead.com";
+```
+
+#### #2: Configure SDK in Android
+Open [`ThunderheadApplication.kt`](https://github.com/thunderheadone/one-sdk-flutter/blob/master/examples/android-add-flutter-to-existing-app-example/app/src/main/java/com/thunderhead/addfluttertoexistingappexample/ThunderheadApplication.kt)
+to initialize the SDK using the integration parameters specific to your ONE environment.
+
+```kotlin
+import com.thunderhead.android.api.oneConfigure
+import com.thunderhead.OneModes;
+// The rest of the imports
+
+class ThunderheadApplication : Application() {
+    override fun onCreate() {
+        super.onCreate()
+        oneConfigure {
+            siteKey = SITE_KEY
+            apiKey = API_KEY
+            sharedSecret = SHARED_SECRET
+            userId = USER_ID
+            host = URI(HOST)
+            touchpoint = URI(TOUCHPOINT)
+            mode = OneModes.USER_MODE
+        }
+    }
+    
+    companion object {
+        const val SITE_KEY = "ONE-XXXXXXXXXX-1022"
+        const val API_KEY = "f713d44a-8af0-4e79-ba7e-xxxxxxxxx"
+        const val SHARED_SECRET = "bb8bacb2-ffc2-4c52-aaf4-xxx"
+        const val USER_ID = "yourUsername@yourCompanyName" // when integrating with Interaction Studio use a numeric user id - see https://eu2.thunderhead.com/one/help/interaction-studio/how-do-i/mobile/one_integrate_mobile_find_integration_info/#username-user-id
+        const val HOST = "https://xx.thunderhead.com"
+        const val TOUCHPOINT = "myAppsNameURI"
+    }
+}
+```
+
+* The configuration parameters include:
+    * `Site Key` (for your specific Space)
+    * `Touchpoint URI`
+        * The native Thunderhead SDK will automatically prefix the URI scheme (i.e. android:// and ios://) when it is omitted. 
+            * i.e. “optimization-example”
+        * If you want to configure under a single Touchpoint, you can explicitly prefix your URI scheme 
+            * i.e. “ionic://optimization-example”
+    * `API Key` & `Shared Secret` (required for OAuth 1.0 authentication)
+    * `Username/User ID` (required for OAuth 1.0 authentication)
+    * `Host name`. 
+        * Typically, this is https://na5.thunderhead.com or https://eu2.thunderhead.com.
+    * `Admin Mode`
+        * Admin mode (adminMode = true) provides you with an interface that lets you add Interaction Points, Activity Capture Points, and Attribute Capture Points to native UI elements within the app.  However, hybrid apps do not support this feature because hybrid solutions typically use  non-native UI elements .  Only preview mode is supported to view your unpublished (In the Works) configuration before publishing it to your live environment.
```

Comment:
please fix spacing

Reply:
The text in this bullet is all in one line.  It looks like a spacing issue in the diff but does not show in the Github README page and is indented properly